config/config_helpers.go

```diff
@@ -632,17 +633,26 @@ func getAWSAccountAlias(ctx *ParsingContext, l log.Logger) (string, error) {
 
 // Return the AWS account id associated to the current set of credentials
 func getAWSAccountID(ctx *ParsingContext, l log.Logger) (string, error) {
+	l.Tracef("Invoking get_aws_account_id()...")
+
 	awsConfig, err := awshelper.CreateAwsConfig(ctx.Context, l, nil, ctx.TerragruntOptions)
 	if err != nil {
+		l.Errorf("get_aws_account_id(): failed to create AWS config: %v", err)
 		return "", err
 	}
 
+	start := time.Now()
```

Comment:
Please remove this timer.

If you want to instrument code to determine durations, it should be done with our OpenTelemetry integration:
https://github.com/gruntwork-io/terragrunt/blob/main/internal/discovery/discovery.go#L970